sos/plugins/postgresql.py

```diff
@@ -35,6 +35,7 @@ class PostgreSQL(Plugin):
     packages = ('postgresql',)
 
     tmp_dir = None
+    filename = 'sos_pgdump.tar'
```

Comment:
How can this work if there is a system copy of Postgres as well as an SCL version (or multiple SCL versions)?

I don't see how we can add this capability and not change the path of the data: I don't see anything in the plugin that prevents the SCL `self.pg_dump()` call from overwriting the work of the call in the main superclass `setup()`? I also don't really know what that is going to do to the collected file records: we are collecting a file twice and it is not the same file!

Also, doesn't that mean that we have these dumps under some bizarre tmp-named path in `/tmp`? Rather than in the `sos_commands/` tree where they belong?

Although the SCL work needs to land soon it seems that this plugin needs major work anyway...